applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -154,6 +221,8 @@ static void cap_start_worker(struct k_work *work)
 	param.count = 0;
 	param.type = BT_CAP_SET_TYPE_AD_HOC;
 
+	k_mutex_lock(&mtx_unicast_group_created, K_FOREVER);
```

Comment:
Does this need to be K_FOREVER? This makes it hard to debug if something goes wrong. Can we set a reasonable timeout and have an ASSERT or error print?